libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -95,12 +95,15 @@ $container-query-breakpoint: 600px;
 
 .main-content {
   display: flex;
-  min-height: utils.line-height('l') * 3; // text content height should always be minimum 3 lines
   box-sizing: border-box;
   flex-direction: column;
   padding: utils.size('xs') 0 utils.size('xxs') utils.size('xxs');
   overflow: hidden;
 
+  &.has-minimum-height {
```

Comment:
I think we should try to fix this in cases where no text is supplied. Right now it looks like this: 
<img width="313" alt="image" src="https://github.com/user-attachments/assets/e5da9298-1e9a-4cd9-bbef-23f35dd7e7e2">

You can probably get around this by adding a couple of classes (`.header-text` and `.body-text` or something) to the two elements where content is rendered  in the end, and adding this to the main-content style rule:
```
&:has(.header-text:empty) {
    padding-block-start: 0;
}

&:has(.body-text:empty) {
    padding-block-end: 0;
}
```